pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+### Purpose
+_Please link to the issue being fixed by this PR:_
+
+- No issue is linked to this PR.
+---
+### Changes
+_Please list out what major changes were made in this PR to address the issue:_
+
+- No changes.
+---
+### Review Instructions
+_Please provide instructions about how should a reviewer test/verify the changes in this PR:_
+
+- No instructions.
+
+---
+### PR Checklist
+_Please ensure the following when opening a PR:_
+
+- [ ] This PR added or updated tests.
+- [ ] This PR updated docstrings or documentation.
+- [ ] This PR applied (WDL/Python) style guidelines, specifically followed NumPy style for this repo.
```

Comment:
I think it would be clearer to say "...specifically followed [NumPy style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html) for this repo" and include the link to an example

Reply:
Sctools doesn't have WDL code right? I think we could take out that part then